src/dotnet/Fable.Core/Import/Node/Querystring.fs

```diff
@@ -0,0 +1,20 @@
+[<AutoOpen>]
+module rec Fable.Import.Node.Querystring
+
+open Fable.Core
+
+type [<AllowNullLiteral>] StringifyOptions =
+    abstract encodeURIComponent: (string -> string) option with get, set
+
+type [<AllowNullLiteral>] ParseOptions =
+    abstract maxKeys: float option with get, set
+    abstract decodeURIComponent: (string -> string) option with get, set
+
+type Globals =
+    abstract stringify: obj: 'T * ?sep: string * ?eq: string * ?options: StringifyOptions -> string
+    abstract parse: str: string * ?sep: string * ?eq: string * ?options: ParseOptions -> 'T
+    abstract escape: str: string -> string
+    abstract unescape: str: string -> string
+
+[<Import("*", "querystring")>] 
+let querystring: Globals = jsNative
```

Comment:
@inosik If I use `module Fable.Import.Node` the F# compiler will complain that there's a module and a namespace (e.g. in `Fable.Import.Node.QueryString`) with the same name in the assembly.